anvil_consortium_manager/models.py

```diff
@@ -72,12 +75,76 @@ def anvil_import(cls, billing_project_name):
         return billing_project
 
 
+class UserEmailEntry(TimeStampedModel, models.Model):
+    """A model to store emails that users could link to their AnVIL account after verification."""
+
+    uuid = models.UUIDField(default=uuid.uuid4)
+    """UUID for use in urls."""
+
+    email = models.EmailField()
+    """The email entered by the user."""
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, on_delete=models.CASCADE
+    )
+    """The user who created the record."""
+
+    date_verification_email_sent = models.DateTimeField()
+    """Most recent date that a verification email was sent."""
+
+    can_be_verified = models.BooleanField(default=True)
```

Comment:
We may not need this field. Instead, it's better to check if an Account exists for the given user or email address. That is likely better because a CC staff member may have added an Account for the email that they are trying to verify.